styleguide/static/styleguide/styleguide.js

```diff
@@ -6,6 +6,20 @@ var toggle = function(el){
 	}
 }
 
+var ripple = function(el){
+	el.onclick = function(event){
+		el.classList.add('run')
+		setTimeout(function(){
+			el.classList.remove('run')
+		}, 1000)
+	}
+}
+
+var ripples = document.getElementsByClassName('ripple');
+for(var i = 0; i < ripples.length; i++) {
```

Comment:
please add a space after `for`

Reply:
also indent with spaces instead of tabs

Reply:
This PR probably falls under not a priority.  I think we can probably move away from this and revisit early next year hopefully.